### PR TITLE
Adds a SLR to the game

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
I'll probably make this an ERT gun for freelancers one day.

## Why It's Good For The Game
Adds a non marine gun that's in the style.

## Changelog
:cl:
add: Added new rifle. Its in req. That's it.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -51. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/magazines/rifles.dm [hunk 1]
```diff
@@ -108,6 +108,20 @@
 	max_rounds = 20
 	gun_type = /obj/item/weapon/gun/rifle/standard_br
 
+//-------------------------------------------------------
+//FAL
+
+/obj/item/ammo_magazine/rifle/slr
+	name = "\improper L21A3 magazine (7.62x51mm)"
+	desc = "A 10mm battle rifle magazine."
```
Comment (XSlayer300) [inline code/modules/projectiles/magazines/rifles.dm:116]:
```suggestion
	desc = "A 7.62mm battle rifle magazine."
```

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Either make it unique or replace an existing gun and give it to someone

**Review by Hughgent — CHANGES_REQUESTED** [review]
Missing a bunch of the on mob sprites

back and suit come to mind.

**Pull request closed without merging**